Explorer/Assets/Scripts/ECS/SceneLifeCycle/Systems/PartitionSceneEntitiesSystem.cs

```diff
@@ -27,10 +27,14 @@ namespace ECS.SceneLifeCycle.Systems
     [UpdateBefore(typeof(ResolveStaticPointersSystem))]
     public partial class PartitionSceneEntitiesSystem : BaseUnityLoopSystem
     {
+        private const int DEPLOYED_SCENES_LIMIT = 90000; // 300x300 scenes (without empty)
```

Comment:
In theory, we can have more than 90k since we already have some scenes outside the -150x150 boundaries. But for now this number shall suffice

Reply:
Yeah, it's a theory only. The main problem was connected to empty scenes